Data-Management/scripts/010_load_catcode.sas

```diff
@@ -0,0 +1,18 @@
+cas casauto;
+
+caslib _ALL_ assign;
```

Comment:
Is the caslib _ALL_ assign necessary of we run the code in batch?
Suggestion to remove it, it slows down the code execution.

Reply:
_caslib _ALL_ assign; is required for the libref statement below.
Code won't run without it.